styles.css

```diff
@@ -0,0 +1,886 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Lato, 'Times New Roman', Times, serif;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  background-color: #e5e5e5;
+  overflow-x: hidden;
+}

```

Comment:
- [ ] Kindly ensure that the content of this container has equal margin-right and margin-left according to the given template. 👍 

## Given Design
<img width="460" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235197972-ea6475cc-8c06-429d-aad4-502af614759b.png">

Reply:
Done.